pkg/kubelet/dockertools/exec.go

```diff
@@ -54,7 +54,8 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 	args = append(args, cmd...)
 	command := exec.Command(nsenter, args...)
 	if tty {
-		p, err := kubecontainer.StartPty(command)
+		var p *os.File
+		p, err = kubecontainer.StartPty(command)
```

Comment:
Sure, though I figured the next time this code was touched would be to delete it anyways 😸